code/public/index.html

```diff
@@ -13,7 +19,11 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Technigo React App</title>
+    <meta property="og:title" content="Tecnigo maze" />
```

Comment:
Nicely done by creating the og tags! Definitely something that I want to try as well :)